aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/dependsOnActionRegistry.js

```diff
@@ -38,12 +39,41 @@
             return action;
         }
 
+        /**
+         * Function to remove symbols that don't match the pattern
+         * @param {string} target - target string
+         * @param {RegExp} pattern -  pattern for correct symbols
+         * */
+        static _replaceForbiddenSymbols(target, pattern) {
+           return target
+               .split('')
+               .map((symbol) => (symbol.search(pattern) !== -1  ? symbol : ''))
+               .join('');
+        }
+
+        /**
+         * Function to develop action's name (allowed symbols a-z, 0-9, -)
+         * @param {string} name - action name
+         * @returns {string} correct action name (according to nameRegex)
+         * */
+        static _getValidName(name) {
+            const nameRegex = /^[a-z0-9-]+$/;
```

Comment:
Literal should be moved as a constant.

Reply:
Note: you could prefer private constant here (out of class, field in bounds of IIFE wrapper)